### DISCUSSION TITLE
Selectable

### DISCUSSION THREAD

**Discussion by lishine** [ariakit/ariakit#768] [Questions] [answered]
I am building a two columns list of products (image + text). It can be selected and then the selected shows additionally in a panel above.
What is the recommended accessible approach?
Maybe like each product will be Tab. Or radio. Or just li.
How the disabled user get to know that something is selected?
Trying component from reakit should I use.
BTW I couldn't add as property to radio , it won't work - switching of state.

**Comment by diegohaz** [comment, marked as answer]
It would be easier to guess with a design or codesandbox. But it looks like a listbox to me:
```jsx
const state = useCompositeState({ orientation: "vertical" });
<Composite {...state} as="ul" role="listbox" aria-label="Products" style={{ columnCount: 2 }}>
  <CompositeItem {...state} as="li" role="option" onFocus={...} onClick={...}>Item</CompositeItem>
  <CompositeItem {...state} as="li" role="option" onFocus={...} onClick={...}>Item</CompositeItem>
  <CompositeItem {...state} as="li" role="option" onFocus={...} onClick={...}>Item</CompositeItem>
  <CompositeItem {...state} as="li" role="option" onFocus={...} onClick={...}>Item</CompositeItem>
  <CompositeItem {...state} as="li" role="option" onFocus={...} onClick={...}>Item</CompositeItem>
</Composite>
```

Or just a list of tabbable items is fine too.

**Reply by lishine** [reply]
Thank you very much
Here is the design
![image](https://user-images.githubusercontent.com/6741645/96360406-c7e3f680-1125-11eb-85c9-4dc18463936f.png)

**Reply by diegohaz** [reply]
Just make them tabbable elements. No composite widget is necessary there. 👍

**Reply by prbxr** [reply]
Thanks.
But How the disabled user get to know that something is selected?
With the focus? But then I should change the state onFocus and not on onClick.

**Reply by diegohaz** [reply]
A listbox would work then.

**Reply by prbxr** [reply]
That actually is infinite list, so each row is wrapped in an element.
So in a row each of the two elements can be selected.
It is like ul->div->li+li
So it will not work simply as a list...

**Comment by lishine** [comment]
And for this one, what would be the best:
(it is selection one out three, essentially like radio, just I could not do AS with radio of reakit)
![image](https://user-images.githubusercontent.com/6741645/96360451-1beedb00-1126-11eb-9fb3-5ec35a89efa8.png)

**Reply by diegohaz** [reply]
I don't understand that design. Is that like desktop icons?

**Reply by lishine** [reply]
In any time one of them or none can be selected and the appropriate panel is shown.
